internal/tag/tag.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package tag
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/acr-cli/acr"
+	"github.com/Azure/acr-cli/cmd/api"
+	"github.com/pkg/errors"
+)
+
+// ListTags will do the http requests and return the digest of all the tags in the selected repository.
+func ListTags(ctx context.Context, acrClient api.AcrCLIClientInterface, repoName string) ([]acr.TagAttributesBase, error) {
```

Comment:
Nit. ListAllTags

Reply:
Changing the method name required me to do renaming in purge cmds and related test files. I'll skip this change to limit the scope of changes to cssc for now.